### PR TITLE
Display retry-after in minutes

### PR DESCRIPTION
**Pull request by chloepouprom**
Display retry-after in minutes (instead of seconds).

Also, increase font size of error messages.

@dannytranlx @mspensieri @spasiu

### CODE REVIEW

**Comment by jugarrit** [comment]
👍

**Comment by dannytranlx** [comment]
👍

File: src/js/reducers/ui-reducer.js [hunk 1]
```diff
@@ -35,7 +35,8 @@ const INITIAL_STATE = {
         wechatChannelDescription: 'To send us a message from WeChat, scan this QR code using the WeChat app.',
         lineChannelDescription: 'To talk to us using LINE, search for our official account using the LINE app and send us a message:',
         smsInvalidNumberError: 'Your phone number isn\'t valid. Please try again.',
-        smsTooManyRequestsError: 'A connection for that number was requested recently. Please try again in {seconds} seconds.',
+        smsTooManyRequestsError: 'A connection for that number was requested recently. Please try again in {minutes} minutes.',
+        smsTooManyRequestsOneMinuteError: 'A connection for that number was requested recently. Please try again in 1 minute',
```
Comment (mspensieri) [inline src/js/reducers/ui-reducer.js]: missing a `.` at the end

**Comment by mspensieri** [comment]
Kind of odd to need two different strings for this case 😕

**Pull request merged**